app/js/components/connection_request_widget.ts

```diff
@@ -39,6 +39,7 @@ class CollectionWidget {
   private initCollectionWidget() {
     const $collectionContainer = $('<div class="collection-entry base-form"></div>');
     const $addEntryButton = $('<button type="button" class="button-small blue add_collection_entry"><i class="fa fa-plus"></i></button>');
+    const $sendButton = $('<button type="submit" id="connection_request_container_send" name="connection_request_container[send]" class="button">Send</button>');
```

Comment:
I would have expected to see a css selector string on the existing send button. And start listening for click events on that button. Not adding a completely new button to the DOM.